misc_scripts/pulsar_project_lists.py

```diff
@@ -22,7 +22,7 @@
 # PSRDB imports
 from tables import *
 from graphql_client import GraphQLClient
-sys.path.append('/fred/oz005/users/acameron/pipeline_stuff/andrew_meerpipe_dev/meerpipe/')
```

Comment:
This path amendment is required to access the db_utils library. Happy to discuss ideas for a more effective way of implementing this, I agree it's not ideal.